iterative_verifier.go

```diff
@@ -137,6 +137,7 @@ type IterativeVerifier struct {
 
 	Tables              []*schema.Table
 	IgnoredTables       []string
+	IgnoredColumns      map[string][]string
```

Comment:
If I remember correctly, we identify columns via `#{table_name}.#{column_name}` in other places (so this could just be a `[]string` instead of a `map[string][]string`). Curious why you chose this approach instead?

Reply:
Where did you find `#{table_name}.#{column_name}`?

The approach here makes skipping tables that have no ignored columns (the vast majority)  O(1). In the alternative approach you suggested you'd have to pair up each table and its columns and iterate through the configured list to check if there are matches